### PR TITLE
SVGMatrix: Describe how to create an instance

### PR DESCRIPTION
**Pull request by jakub-trzebiatowski**
#### Summary
I've described how to create an instance of `SVGMatrix`.

#### Motivation
The documentations lacks a description of a constructor (as it's not public, by the spec), so the MDN reader doesn't immediately know how to create an instance of `SVGMatrix`.

#### Supporting details

https://www.w3.org/TR/SVG11/coords.html#InterfaceSVGMatrix

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by cubuspl42** [comment]
Unfortunately, although `DOMMatrix` is replaced by `DOMMatrix`, the latter is not fully supported by latest Chrome. It makes expanding `SVGMatrix`documentation more justified.

**Comment by cubuspl42** [comment]
> ## Flaws
> URL: [`/en-US/docs/Web/API/SVGMatrix`](https://pr10409.content.dev.mdn.mozit.cloud/en-US/docs/Web/API/SVGMatrix) Title: `SVGMatrix` [on GitHub](https://github.com/mdn/content/blob/10409/merge/files/en-us/web/api/svgmatrix/index.md) Flaw count: 17
> 
> * **macros**:
>   
>   * `/en-US/docs/Web/API/SVGMatrix/a does not exist`
>   * `/en-US/docs/Web/API/SVGMatrix/b does not exist`
>   * `/en-US/docs/Web/API/SVGMatrix/c does not exist`
>   * `/en-US/docs/Web/API/SVGMatrix/d does not exist`
>   * `/en-US/docs/Web/API/SVGMatrix/e does not exist`
>   * _and 12 more flaws omitted_

These flaws were already present in the document, as I understand it.

File: files/en-us/web/api/svgmatrix/index.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ which, when expanded into a 3x3 matrix for the purposes of matrix arithmetic, be
 
 An **`SVGMatrix`** object can be designated as read only, which means that attempts to modify the object will result in an exception being thrown.
 
+**`SVGMatrix`** does not have a public constructor. You can create a **`SVGMatrix`** through `createSVGMatrix` method on [`SVGSVGElement`](/en-US/docs/Web/API/SVGSVGElement#methods).
```
Comment (wbamberg) [inline files/en-us/web/api/svgmatrix/index.md:31]:
```suggestion
**`SVGMatrix`** does not have a public constructor. You can create an **`SVGMatrix`** using the `createSVGMatrix()` method of [`SVGSVGElement`](/en-US/docs/Web/API/SVGSVGElement#methods).
```

**Review by wbamberg — APPROVED** [review]
Thanks for your contribution @cubuspl42 !

**Pull request merged**